### ISSUE TITLE
NotFoundError: Failed to execute 'removeChild' on 'Node': The node to be removed is not a child of this node.

### ISSUE THREAD

**Issue by MartijnHols** [WoWAnalyzer/WoWAnalyzer#948]
Private: https://sentry.io/wowanalyzer/wowanalyzer-app/issues/414962047/
Public: https://sentry.io/share/issue/3f525206057d4e36a3ed065bfaefd3f9/

```
NotFoundError: Failed to execute 'removeChild' on 'Node': The node to be removed is not a child of this node.
  at N (../node_modules/react-dom/cjs/react-dom.production.min.js:222:251)
  at g (../node_modules/react-dom/cjs/react-dom.production.min.js:144:441)
  at Bg (../node_modules/react-dom/cjs/react-dom.production.min.js:148:97)
  at b (../node_modules/react-dom/cjs/react-dom.production.min.js:158:10)
  at m (../node_modules/react-dom/cjs/react-dom.production.min.js:169:88)
...
(11 additional frame(s) were not displayed)
```

(there are a lot of events for this in Sentry)

**Comment by MartijnHols** [comment]
I tried to reproduce this many times by following the steps the user took as shown in Sentry to no avail. The line triggering the error also doesn't help.

I started figuring it's very likely an extension from the user such as an Adblocker removing a React element but since we don't actually do any ads it would be weird if that was it but some filter might be much too aggressive, no way of knowing for sure. After some more Googling just now (and a bit of luck) I think this is more likely the cause of the crash: https://github.com/facebook/react/issues/11538

Not sure yet how to fix.
